README.md

```diff
@@ -146,6 +163,159 @@ helm_chart(
+
+Example of use:
+```python
+# partial sample of chart with matching values
+helm_chart(
+    name = "dogchart",
+    ...
+)
+
+sops_decrypt(
+    name = "dogpark_secrets",
+
+    srcs = [":secrets.yaml"],
+    sops_yaml = ":sops-dogpark.yaml",
+
+    tags = ["manual"],
+)
+
+
+# partial sample of release with matching values
+helm_release(
+    name="release",
+
+    chart = ":dogchart",
+    helm_version="v3",
+    release_name = "dogparkservice",
+    values_yaml = [ ":dogpark_secrets", ":values.yaml" ],
+    kubernetes_context = "prod",
+    ...
+)
+
+# full example of diff
+helm_diff(
+    name = "dogdiff",
+    chart = ":dogchart",
+    helm_version="v3",
+    release_name = "dogparkservice",
+    secrets_yaml = ":secrets.yaml",
+    sops_yaml = ":sops-dogpark.yaml",
+    values_yaml = [ ":dogpark_secrets", ":values.yaml" ],
+    kubernetes_context = "prod",
+
+    tags = ["manual"],
+)
+```
+
+This markup would allow a fictional `bazel run //path/prod:release` before a change to be followed
+by a `bazel run //path/prod:dogdiff` (although in practice I just use "diff" here, but I've expanded
+that to be more explicit).  Notice that many of the `helm_release()` attributes are echoed to
+`helm_diff()` and that both `helm_diff` and `helm_release` may need a `--action_env=HOME` so that
+the underlying `helm` can find your ${HOME}/.kube/config to find the target cluster.
+
+The following attributes are accepted by the rule (some of them are mandatory).  Note that some
+will mark these `tags = ["manual"]` to cause the rule to only be used when explicitly stated rather
+than a wildcard.  This can be especially helpful when some developers lack SOPS permission on
+sensitive enviroments (ie prod).
+
+|  Attribute | Mandatory| Default | Notes |
+| ---------- | --- | ------ | -------------- |
+| chart | yes | - | Chart package (targz). Must be a label that specifies where the helm package file (Chart.yaml) is. It accepts the path of the targz file (that bazel will resolve to the file) or the label to a target rule that generates a helm package as output (`helm_chart` rule). |
+| namespace | false | default | Namespace name literal where this release is installed to. It supports the use of `stamp_variables`. |
+| release_name | yes | - | Name of the Helm release. It supports the use of `stamp_variables`|
+| values_yaml | no | - | Several values files can be passed similar to a `helm_release()` -- for an accurate comparison, the same values files as the `helm_release()`. |
+| helm_version | no | "" | Force the use of helm v3 to deploy the release. Can be used to try v2. |
+| kubernetes_context | no | "" | Context of kubernetes cluster |
+
+
+### helm_lint
+
+`helm_lint` is a convenience function to stage the given chart for a `helm lint`; this simply makes it easier to do the "helm lint" of a chart, with the goal of reducing the burden to linting charts.  This is useful to avoid certain basic errors.
```

Comment:
```suggestion
`helm_lint` is a convenience function to stage the given chart for a `helm lint`; this simply makes it easier to do the "helm lint" of a chart, with the goal of reducing the burden to linting charts.  This is useful to avoid certain basic errors.
To clarify, this linting is done to the derived helm chart archive, not to any release, which means it should be possible without permissions to view/edit an actual deployment.
```